fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -997,8 +1000,11 @@ public TTableDescriptor toThrift() {
     }
 
     @Override
-    public AnalysisJob createAnalysisJob(AnalysisJobScheduler scheduler, AnalysisJobInfo info) {
-        return new AnalysisJob(scheduler, info);
+    public BaseAnalysisTask createAnalysisTask(AnalysisTaskScheduler scheduler, AnalysisTaskInfo info) {
```

Comment:
what the relation bewteen job and task?

Reply:
one analyzestmt map to one job map to multiple task